AudioKit/Common/Nodes/Playback/Player/ClipPlayer/AKClipPlayer.swift

```diff
@@ -216,8 +186,36 @@ open class AKClipPlayer: AKNode, AKTiming {
         set { playerNode.pan = newValue }
     }
 }
-extension AKClipPlayer {
+extension AKClipPlayer: AKTiming {
```

Comment:
@ryanfrancesconi Moving the AKTiming functions to an extension makes it clear that the functions are there to satisfy AKTiming.  Maybe AKPlayer could do the same.